src/components/ActionButtonBar.vue

```diff
@@ -62,6 +62,15 @@ import { Model } from "survey-vue";
 @Component
 export default class ActionButtonBar extends Vue {
   @Prop() survey?: Model;
+  startModal() {
+    let text =
+      this.$i18n.locale === "fr"
+        ? "Cette action effacera toutes les données non enregistrées. Êtes-vous sûr de vouloir continuer ?"
```

Comment:
I don't think https://github.com/canada-ca/aia-eia-js/blob/master/src/survey-enfr.json is the right spot, but it seems like all these hardcoded translation strings should be external lookups

Reply:
Good point. As this is only applied to this button, not used anywhere else, I keep it in it's own file to make it simpler.